docs/src/lib/types.md

```diff
@@ -57,6 +57,9 @@ The `RepeatedVector` and `StackedVector` types are subtypes of `AbstractVector`
 with the exception that they are read only. Note that they are not exported and should not be constructed directly,
 but they are columns of a `DataFrame` returned by `stack` with `view=true`.
 
+The `Row` type is a special type used for selection operations to signal that the wrapped function should be applied
```

Comment:
`ByRow` sounds OK. Not sure what are the advantages or drawbacks compared to `EachRow`.

Reply:
I definitely do not understand all the subtleties here, but I would propose something along the lines of `Map(f)(args...) = map(f, args...)`. `Map(f)` read nicely, and as @nalimilan  points out it could be useful outsides of DataFrames.

Reply:
I am OK with `Map`, @nalimilan - what do you think?

The problem is that for the purposes of DataFrames.jl it would have to be:
```
Map(f)(v::AbstractVector) = map(f, v)
Map(f)(nt::NamedTuple{N, <:Tuple{Vararg{AbstractVector}}} where N) = map(f, Tables.namedtupleiterator(nt))
```
which does not sound very generic.

If `Map` is not accepted then `ByRow` is simply shorter than `EachRow` which I think is relevant here + as I have said it is used in DataFramesMeta.jl so it would be more consistent.

Reply:
Ah, I see. So it is more of a `Map(f)(v) = map(f, Tables.rows(v))`, but `MapRows` is not such a nice name. Now I see the point of `ByRow` a bit more clearly.

Reply:
Exactly (except that `Tables.rows` does not guarantee to return `NamedTuple`s, and `Tables.namedtupleiterator` does and it will be exposed soon as a public API).